vue/components/personalization.vue

```diff
@@ -149,10 +149,7 @@ export const personalization = {
         state: {
             handler: function(state, previousState) {
                 if (this.diffInitialsExtra(state.initialsExtra, previousState.initialsExtra)) {
-                    this.$bus.trigger(
-                        "initials_change",
-                        this.sanitizeInitialsExtra(state.initialsExtra)
-                    );
+                    this.$bus.trigger("initials_change", state.initialsExtra);

```

Comment:
why this change? we should have proper initials in the event right away.

Reply:
Yup, I've just analyzed this, there's a huge difference in semantics. This is not a proper fix.

Reply:
Because of the comment below. If this is the event for a UI-level change then the sanitisation shouldn't happen here